tools/firtool/firtool.cpp

```diff
@@ -773,6 +779,15 @@ static LogicalResult processBuffer(
         firrtl::createMergeConnectionsPass(
             !disableAggressiveMergeConnections.getValue()));
 
+  if (stripMuxPragmas) {
+    if (addMuxPragmas) {
+      emitError(module->getLoc())
+          << "--strip-mux-pragmas and --add-mux-pragmas are conflicting.";
+      return failure();
+    }
+    addMuxPragmas = false;
```

Comment:
Actually, this line seems redundant, being dominated by check of `addMuxPragmas` so it's always already non-true.  Can this all be reduced to `if (stripMuxPragmas && addMuxPragmas) return emitError(...)` (perhaps `==` instead of `&&`)?

Reply:
That makes sense to me, changed to use `==`.